types/Items.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface ItemData {
+  items: Item[]
+}
+
+type Item = {
```

Comment:
```suggestion
export type Item = {
```

あと item というワードが抽象的すぎるので名前を変えて、その上で export しておいてほしいかもです

Reply:
確かにexportの場合もう少し命名しっかりしないとですね。他にもitemsが多くありました。

こちら c7199eb で個別にしたことでitemでも問題なさそうになったので、一旦このままにしています。
よろしくお願いいたします。